legend-engine-language-pure-grammar/src/test/java/org/finos/legend/engine/language/pure/grammar/test/roundtrip/TestMappingGrammarRoundtrip.java

```diff
@@ -442,4 +442,28 @@ public void testAggregationAware()
                 "  }\n" +
                 ")\n");
     }
+
+    @Test
+    public void testMappingWithIncludedMapping()
+    {
+        test("###Mapping\n" +
+                "Mapping test::decomposeMapping\n" +
+                "(\n" +
```

Comment:
For this example to make sense, shouldn't the source mapping refer to dbInc? And shouldn't the 'target' mapping have a reference to db?

Reply:
you're right! Thanks for pointing that out, I updated and moved this test to TestRelationalGrammarRoundtrip